### PR TITLE
Edit pass

### PR DESCRIPTION
**Pull request by grahamgower**

### CODE REVIEW

File: adding_species_manuscript.tex [hunk 1]
```diff
@@ -430,7 +430,7 @@ \section*{Additions to \stdpopsim}
 
 Another important extension of \stdpopsim allows augmenting a genome assembly by genome annotations, such as coding regions, promoters, conserved elements, etc.
 These annotations can be used to simulate selection at a subset of sites (e.g., the annotated coding regions)
-using parametric distribution(s) of fitness effects.
+using a parametric distribution of fitness effects.
```
Comment (igronau) [inline adding_species_manuscript.tex]: I think that the plural was used here on purpose to indicate that different annotated sets can be associated with different DFEs. Maybe we should use the plural and just remove the parentheses around the s?

**Review by igronau — COMMENTED** [review]
Thanks, @grahamgower. I have only one comment about your suggested edit about the DFEs. Can you have another look there?

File: adding_species_manuscript.tex [hunk 2]
```diff
@@ -205,8 +205,8 @@ \section*{Introduction}
 such as inbreeding or selfing \citep{Blischak2020}, skewed offspring
 distributions \citep{Montano2016}, and intense artificial selection \citep{MacLeod2013, MacLeod2014}.
 
-Simulations can be useful at all stages of this work --
-for planning studies, analyzing data, testing inference methods,
+Simulations can be useful at all stages of this work---for
```
Comment (lauterbur) [inline adding_species_manuscript.tex:208]: Why three dashes here? Is this a latex thing?

File: adding_species_manuscript.tex (hunk 1 above)
Comment (grahamgower) [reply adding_species_manuscript.tex]: Sure, happy to go with the plural instead.

File: adding_species_manuscript.tex (hunk 2 above)
Comment (grahamgower) [reply adding_species_manuscript.tex:208]: An em-dash is three hyphens, `---`, while just two gives an en-dash. https://www.scribbr.com/language-rules/dashes/

**Comment by lauterbur** [comment]
Gotcha, thanks!

**Pull request merged**